lib/ebanx/version.rb

```diff
@@ -1,3 +1,3 @@
 module Ebanx
-  VERSION = '1.6.0'
+  VERSION = '1.7.0'
```

Comment:
I guess It could be a patch

```suggestion
  VERSION = '1.6.1'
```